_includes/scenarios/integrate/prepare-enmeshed-onboarding-package.md

```diff
@@ -0,0 +1,132 @@
+Communication and sharing of information between a Connector and another Identity requires the existence of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) between them. This guide describes how to create an onboarding package on a Connector that can then be used by other Identities to send a Relationship Request to the Connector. The creation of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) is the first required step in this process.<!--- Fundamental to this is an understanding of how to create a [Relationship Template]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).--->
+
+## Create a RelationshipTemplate
+
+A RelationshipTemplate is a formal description of the aspects of a Relationship that can be established between two Identities. In particular, it can specify [Requests]({% link _docs_integrate/data-model-overview.md %}#request) sent from the one Identity to the other Identity, which must be accepted as a prerequisite for the establishment of the Relationship. <!--- For example, you can decide what data should be exchanged between the two Identities at the time the Relationship is established.-->
+
+### Send Request
+
+To create a RelationshipTemplate on your Connector, you need to send a Request by calling `POST /api/v2/RelationshipTemplates/Own` with the following JSON payload:
```

Comment:
http request (otherwise confusing with the enmeshed Request you used and linked in the paragraph above). However, ideally you could link to a usecase here (and also later).

Reply:
I will change the naming for requests to http requests in these cases to make it more clear.
I will change the position of the links to the use cases, so that they are in a more prominent place.